pkg/controller/migmigration/restore.go

```diff
@@ -222,8 +224,25 @@ func (t *Task) buildRestore(backupName string) (*velero.Restore, error) {
 // Update a Restore as desired for the destination cluster.
 func (t *Task) updateRestore(restore *velero.Restore, backupName string) {
 	restorePVs := true
+
+	namespaceMapping := make(map[string]string)
+	for _, namespace := range t.namespaces() {
+		mapping := strings.Split(namespace, ":")
+		if len(mapping) == 1 {
+			namespaceMapping[namespace] = namespace
+		}
+		if len(mapping) == 2 {
+			if mapping[1] == "" {
+				namespaceMapping[mapping[0]] = mapping[0]
+			} else {
+				namespaceMapping[mapping[0]] = mapping[1]
+			}
+		}
+	}
+
```

Comment:
Not sure about this behaviour, but above snippet always passes mapping to restore. 
For example: 
 ```
namespaces:
    - foo1:bar1
    - bar2
``` 
becomes
`{"foo1":"bar1", "bar2":"bar2"}`

Reply:
See the above comment, but I think the appropriate value is actually `{"foo1":"bar1"}`